tensorflow_addons/image/testing_imageops.ipynb

```diff
@@ -0,0 +1,728 @@
+{
```

Comment:
<p>Please update this to say something like "Here is the list of image operations we'll be covering in this example"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>